tests/ops/fixtures/saas/auth0_fixtures.py

```diff
@@ -0,0 +1,173 @@
+        "access_token": pydash.get(saas_config, "auth0.access_token")
+        or secrets["access_token"],
+        "client_id": pydash.get(saas_config, "auth0.client_id") or secrets["client_id"],
+    }
+
+
+@pytest.fixture(scope="function")
+def auth0_identity_email():
+    return pydash.get(saas_config, "auth0.identity_email") or secrets["identity_email"]
+
+
+@pytest.fixture
+def auth0_config() -> Dict[str, Any]:
+    return load_config("data/saas/config/auth0_config.yml")
+
+
+@pytest.fixture
+def auth0_dataset() -> Dict[str, Any]:
+    return load_dataset("data/saas/dataset/auth0_dataset.yml")[0]
+
+
+@pytest.fixture(scope="function")
+def auth0_connection_config(db: session, auth0_config, auth0_secrets) -> Generator:
+    fides_key = auth0_config["fides_key"]
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "key": fides_key,
+            "name": fides_key,
+            "connection_type": ConnectionType.saas,
+            "access": AccessLevel.write,
+            "secrets": auth0_secrets,
+            "saas_config": auth0_config,
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def auth0_dataset_config(
+    db: Session,
+    auth0_connection_config: ConnectionConfig,
+    auth0_dataset: Dict[str, Any],
+) -> Generator:
+    fides_key = auth0_dataset["fides_key"]
+    auth0_connection_config.name = fides_key
+    auth0_connection_config.key = fides_key
+    auth0_connection_config.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": auth0_connection_config.id,
+            "fides_key": fides_key,
+            "dataset": auth0_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def auth0_erasure_data(
+    auth0_connection_config, auth0_erasure_identity_email, auth0_secrets
+) -> Generator:
+    """
+    Creates a dynamic test data record for erasure tests.
+    Yields user ID as this may be useful to have in test scenarios
+    """
+
+    base_url = f"https://{auth0_secrets['domain']}"
+    # Create user
+    body = {
+        "client_id": auth0_secrets["client_id"],
+        "email": auth0_erasure_identity_email,
+        "password": "p@sswordtest1",
+        "connection": "Username-Password-Authentication",
+        "username": "testaccount",
+        "given_name": "test",
+        "family_name": "account",
+        "name": "test account",
+        "nickname": "test",
+        "blocked": False,
+        "email_verified": True,
+        "phone_number": "+012345678",
+        "phone_verified": True,
+        "verify_phone_number": True,
+        "verify_password": True,
+    }
+    headers = {"Authorization": f"Bearer {auth0_secrets['access_token']}"}
+    users_response = requests.post(
+        url=f"{base_url}/dbconnections/signup", json=body, headers=headers
```

Comment:
Why are we using this `/dbconnections` endpoint instead of something like [/api/v2/users](https://auth0.com/docs/api/management/v2/#!/Users/post_users)?

Reply:
Both endpoints can be used to create user but I have updated code to use /api/v2/users